e2e/content-services/metadata/metadata-properties.e2e.ts

```diff
@@ -168,8 +160,6 @@ describe('CardView Component - properties', () => {
         await viewerPage.checkInfoSideBarIsDisplayed();
         await metadataViewPage.clickOnPropertiesTab();
 
-        await metadataViewPage.informationButtonIsDisplayed();
-
         await CheckboxPage.uncheck(metadataViewPage.defaultPropertiesSwitch);
 
         await metadataViewPage.informationButtonIsNotDisplayed();
```

Comment:
This method does not exist on the type. Please check the screenshot.

![Screenshot from 2023-09-02 13-13-56](https://github.com/Alfresco/alfresco-ng2-components/assets/111955585/8318f2e3-f9a0-4c56-8124-5030331a0847)